src/WebJobs.Extensions.DurableTask/AzureStorageDurabilityProviderFactory.cs

```diff
@@ -41,6 +41,8 @@ internal class AzureStorageDurabilityProviderFactory : IDurabilityProviderFactor
             this.defaultConnectionName = this.azureStorageOptions.ConnectionStringName ?? ConnectionStringNames.Storage;
         }
 
+        public string Name => "Azure Storage";
```

Comment:
It feels a little strange having spaces in a symbolic name like this that needs to be referenced in configuration. I wonder if we should use "AzureStorage" instead of "Azure Storage". I fear that customers may accidentally choose the former when trying to be explicit in their configuration.

Reply:
Definitely agree on having types with no space characters. Feels more inline with standard configuration enumerations.